### PR TITLE
prepared database, added student models and relationship between stud…

### PR DESCRIPTION
**Pull request by nguyen-tuongquy**
…ent and user

### CODE REVIEW

**Comment by nguyen-tuongquy** [comment]
ready

File: app/Student.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Student extends Model
+{
+    protected $fillable = [
```
Comment (quoctd-0607) [inline app/Student.php:9]: protected $fillable = [
        'fullname', 
         'dob',
....
    ];

File: app/Student.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        'fullname', 'dob', 'student_no', 'user_id', 'gender', 'address', 
+    ];
+
+    public function user()
+    {
+        return $this->belongsTo(User::class, 'user_id');
+    }
+}
```
Comment (quoctd-0607) [inline app/Student.php:17]: thêm end line vào

File: app/User.php [hunk 3]
```diff
@@ -16,7 +16,7 @@ class User extends Authenticatable
      * @var array
      */
     protected $fillable = [
-        'name', 'email', 'password',
+        'name', 'email', 'password', 'phone',
```
Comment (quoctd-0607) [inline app/User.php:19]: mỗi phần tử trên 1 line

**Comment by nguyen-tuongquy** [comment]
ready

**Comment by quoctd-0607** [comment]
merged

**Pull request merged**